static-build/components/TestCard/styles.css

```
@@ -153,9 +153,13 @@
 
 /* visually hide bundler names */
 .iconList figcaption {
-  width: 0;
-  height: 0;
-  text-indent: -99999999px;
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
```

```suggestion
  width: 0;
  height: 0;
  overflow: hidden;
```

I agree with @wangxianzhu, this seems to work fine.